includes/class-wp-job-manager-recaptcha.php

```diff
@@ -0,0 +1,242 @@
+<?php
+/**
+ * File containing the WP_Job_Manager_Recaptcha class.
+ *
+ * @package wp-job-manager
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Notices_Conditions_Checker class.
+ *
+ * @since 1.40.0
+ * @internal
+ */
+class WP_Job_Manager_Recaptcha {
```

Comment:
We probably want to put this class under the namespace `WP_Job_Manager`

Reply:
done in cb4210dfe6288eb66c8c715897b1f9ec794e855f